.ci/scripts/update_ci_branches.py

```diff
@@ -13,7 +20,7 @@
 
 for branch in branches:
     print(f"Updating {branch}")
-    if type(branch) == float and branch < 3.23 :
+    if isfloat(branch) and float(branch) < 3.23 :
```

Comment:
Oh, so this is yaml turning `1.2` into floats, but `1.2.3` are implicitly strings? Now when some one declares a version rightfully as `"2.3"` we get a string, but your `isfloat` is still able to cast and returns `True`. Right?

Can we `pkg_resource.version.parse` to compare them? Also Why should we cap this at 3.23? All branches i see configured are less than that, and once we add 3.23 to pulpcore important branches we want to publish images, no?

Reply:
BTW the language code for Norway equals false. No good, Yaml!

Reply:
Thanks for the clarification. But the float comparison is still not what we want.
`"3.123" > "3.23"` while `3.123 < 3.23`.
Versions are really not like numbers.

Reply:
https://github.com/pulp/pulp-oci-images/pull/455